webapp/src/container.tsx

```diff
@@ -349,7 +355,7 @@ export class SettingsMenu extends data.Component<SettingsMenuProps, SettingsMenu
             {
                 // we always need a way to clear local storage, regardless if signed in or not
             }
-            {targetTheme.feedbackUrl ? <a className="ui item" href={targetTheme.feedbackUrl} role="menuitem" title={lf("Give Feedback")} target="_blank" rel="noopener noreferrer" >{lf("Give Feedback")}</a> : undefined}
```

Comment:
do we want to get rid of this? is this the URL we use for partner feedback portals (like in minecraft/micro:bit)?

Reply:
As far as I could tell, no. The only places where `feedbackUrl` is used is not from the appTheme. Across all the workspaces, there are instances of `feedbackUrl` for experiments and code cards but that's separate. The feedback url from the appTheme should only be used for cases in the settings menu which should all get replaced with OCV. This feedback option is also not in the settings menu on live, so I think it's safe to get rid of. If there are still concerns about it, I can keep it in, but it should be safe.

One thing I forgot to do was remove `feedbackUrl` from the appTheme interface in `pxtarget.d.ts`, I was planning on doing removing it, but I can also just leave it in for posterity.